src/language-js/printer-estree.js

```diff
@@ -4486,41 +4487,26 @@ function printArgumentsList(path, options, print) {
 
     const somePrintedArgumentsWillBreak = printedArguments.some(willBreak);
 
-    // [prettierx merge from prettier@1.19.0] (...)
-    // [prettierx] simpleConcat not used:
-    // const simpleConcat = concat(["(", concat(printedExpanded), ")"]);
+    // [prettierx] parenSpace option support (...)
+    const simpleConcat = concat([
+      "(",
+      parenSpace,
+      concat(printedExpanded),
+      lastArgAddedLine ? "" : parenSpace,
+      ")",
```

Comment:
needed at this point to resolve lint issue:

```suggestion
      ")"
```